src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -111,6 +168,65 @@
                     "description": "Milliseconds since last update from peer"
                 }
             }
+        },
+        "self_test_report": {
+            "id": "self_test_report",
+            "description": "Self test single test report",
+            "properties": {
+                "disk_test_stoptime": {
```

Comment:
I'm confused about stoptime -- elsewhere in the code it looks like the test is intentionally waiting for stoptime (like it's a target runtime), but here it seems to be described as part of the report like it's a variable runtime

Reply:
Yeah this is stub code to be removed and replaced with the actual result types from the respective disk + network benchmarks

Reply:
I should/will document this in the property description here